coderd/httpmw/ratelimit.go

```diff
@@ -22,14 +25,37 @@ func RateLimitPerMinute(count int) func(http.Handler) http.Handler {
 	}
 	return httprate.Limit(
 		count,
-		1*time.Minute,
+		window,
 		httprate.WithKeyFuncs(func(r *http.Request) (string, error) {
 			// Prioritize by user, but fallback to IP.
 			apiKey, ok := r.Context().Value(apiKeyContextKey{}).(database.APIKey)
-			if ok {
+			if !ok {
+				return httprate.KeyByIP(r)
+			}
+
+			if r.Header.Get(codersdk.BypassRatelimitHeader) == "" {
```

Comment:
Do you care if the value of the header is `false`? Should you `strconv.ParseBool`?